storage/engine/rocksdb_test.go

```diff
@@ -493,3 +493,77 @@ func BenchmarkMVCCMergeTimeSeries(b *testing.B) {
 	}
 	runMVCCMerge(&value, 1024, b)
 }
+
+// runMVCCComputeStats merges value into numKeys separate keys.
+func runMVCCComputeStats(numVersions, valueBytes int, b *testing.B) {
+	const rangeBytes = 64 * 1024 * 1024
+	numKeys := rangeBytes / (valueBytes * ((numVersions / 2) + 1))
+	rocksdb, stopper := setupMVCCScanData(numVersions, numKeys, valueBytes, b)
+	defer stopper.Stop()
+
+	prewarmCache(rocksdb)
+
+	b.SetBytes(rangeBytes)
+	b.ResetTimer()
+
+	b.RunParallel(func(pb *testing.PB) {
```

Comment:
Probably not that useful to allow parallelism here. I'd replace this the non-parallel:

```
for i := 0; i < b.N; i++ {
}
```